features/support/env.rb

```diff
@@ -2,6 +2,16 @@
 # Set this explicitly
 $api_key = "a35a2a72bd230ac0aa0f52715bbdc6aa"
 
+AfterConfiguration do |_config|
+  MazeRunner.config.receive_no_requests_wait = 10 if MazeRunner.config.respond_to? :receive_no_requests_wait=
+  # TODO: Remove once the Bugsnag-Integrity header has been implemented
```

Comment:
It'd be good to create a ticket to track this when this functionality is merged, so that this doesn't get left in.

Reply:
PLAT-5510.